fighters/link/src/acmd/smashes.rs

```diff
@@ -1,17 +1,51 @@
 
 use super::*;
 
+#[acmd_script( agent = "link", script = "game_attacks4" , category = ACMD_GAME , low_priority)]
+unsafe fn game_attacks4 (fighter: &mut L2CAgentBase) {
+	let lua_state = fighter.lua_state_agent;
+	let boma = fighter.boma();
+	frame(lua_state, 10.0);
+	if is_excute(fighter) {
+		WorkModule::on_flag(fighter.module_accessor, /*Flag*/ *FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD);
+	}
+	FT_MOTION_RATE(fighter, 1.5);
```

Comment:
lets change this to the standard `FT_MOTION_RATE` format `FT_MOTION_RATE(fighter, C / (B - A))`